src/main/java/com/nextstep/lotto/lotto/Lotto.java

```diff
@@ -0,0 +1,49 @@
+package com.nextstep.lotto.lotto;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+public class Lotto {
+
+	public static final int LOTTO_SIZE = 6;
+	private static final String LOTTO_NUMBER_CAN_ONLY_6_DIGITS = "로또의 숫자는 6개만 가능합니다.";
+	private final Set<LottoNumber> numbers;
+
+	public Lotto(LottoNumberGenerator lottoNumberGenerator) {
```

Comment:
파라미터로 generator를 전달 받으면 의존성이 생기게 됩니다.

```suggestion
	public Lotto(Set<LottoNumber> numbers) {
```

Reply:
넵! Set으로 전달받도록 수정하였습니다!